_includes/skill_table.md

```diff
@@ -0,0 +1,122 @@
+{%- assign default_language = site.defaults[0].values.lang -%}
+{%- assign authors = site.authors | where: "lang", default_language -%}
+
+{%- assign langs_framework_misc = "langs,frameworks,misc" | split: "," -%}
+{%- assign self_want = "self,want" | split: "," -%}
+
+{% for stype in langs_framework_misc -%}
+
+<h1> {{ site.data.skillset["titles"][stype] }} </h1>
+
+    {%- for skill_ in site.data.skillset[stype] -%}
+        {%- assign skill_name = skill_[0] -%} {%- comment -%} skill name we are searching for {%- endcomment -%}
+        {%- assign skill = skill_[1] -%} {%- comment -%} possible skill-related skills {%- endcomment -%}
```

Comment:
"skill related skills"?

Reply:
Ahhhh, how should I phrase that? I made an awful job naming variables here...

Reply:
`skill_[1]` is the "vers" variable, right? So maybeeee "possible multiple versions of this skill"? But I'm not sure